odoo-shippable/scripts/library.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+git_clone_copy(){
```

Comment:
Yeah, it does.
![captura de pantalla de 2016-06-22 19-47-38](https://cloud.githubusercontent.com/assets/324683/16287583/1e3f4958-38ae-11e6-847e-bf0241579239.png)